front/src/utils/date.ts

```diff
@@ -31,6 +31,23 @@ export function dateTimeFormatting(date: Date, withoutTime: boolean = false) {
   return dateToUTC.tz(dayjs.tz.guess()).format(dateFormat).replace(/\./gi, '');
 }
 
+export function formatDay(locale = 'fr') {
+  if (locale === 'en') {
+    dayjs.locale('en');
+  } else if (locale === 'fr') {
+    dayjs.locale('fr');
+  }
```

Comment:
You should throw if the locale is not 'en' or 'fr'
```suggestion
  if (!['en', 'fr'].includes(locale)) {
    throw new Error('Invalid locale');
  }
  dayjs.locale(locale);
```

Reply:
done